ui/src/components/setlist/SetlistEditorContainer.tsx

```diff
@@ -149,160 +241,123 @@ const SetlistEditorContainer: FC<SetlistEditorProps> = ({ actionOnEditor }) => {
         </Snackbar>
 
         <form onSubmit={handleSubmit(handleSaveSetlist)}>
-          <Toolbar>
-            {editorMode(actionOnEditor)}
-            <Button
-              type={'submit'}
-              color="secondary"
-              variant="contained"
-              sx={{
-                mr: 1,
-                textTransform: 'none',
-                borderRadius: '100px',
-                px: 3,
-              }}
-            >
-              Save
-            </Button>
-            <Button
-              color={'secondary'}
-              sx={{
-                textTransform: 'none',
-                borderRadius: '100px',
-                border: 1,
-                px: 2,
-              }}
-              onClick={() => navigate('/setlist')}
-            >
-              Cancel
-            </Button>
-          </Toolbar>
-
-          <Stack direction="row" spacing={10}>
+          <PageHeader
+            title={action === 'edit' ? 'Edit Setlist' : 'New Setlist'}
+            icon={<QueueMusic />}
+            actionButtons={
+              <Stack direction="row">
+                <Button
+                  type={'submit'}
+                  color="secondary"
+                  variant="contained"
+                  sx={{
+                    mr: 1,
+                    textTransform: 'none',
+                    borderRadius: '100px',
+                    px: 3,
+                  }}
+                >
+                  Save
+                </Button>
+                <Button
+                  color={'secondary'}
+                  sx={{
+                    textTransform: 'none',
+                    borderRadius: '100px',
+                    border: 1,
+                    px: 2,
+                  }}
+                  onClick={() => navigate('/sosetlistng')}
```

Comment:
where this supposed to lead to ya

Reply:
fixed that in the new setlist issue haha